app/services/updaters/group_updater.rb

```diff
@@ -39,6 +39,7 @@ def process_params(params)
     @pending_users = { add: users(:member_ids), remove: users(:remove_ids) }
     delete_user_id_keys_from_params
     delete_leader_id_if_empty
+    delete_size_if_empty
```

Comment:
I think I'd prefer to simply hide the field on the form if the group (or its size) is locked - that seems like it makes more sense to me as opposed to passing empty params.